keyboards/mwstudio/mw660/readme.md

```diff
@@ -0,0 +1,22 @@
+# MW660
+
+![MW660](https://i.imgur.com/kJx0mpi.png)
```

Comment:
This uses a smaller but still high res image that is more suitable for github markdown.
```suggestion
![MW660](https://i.imgur.com/kJx0mpih.png)
```